tools/codegen/cmd/featuregate-test-analyzer.go

```diff
@@ -582,13 +582,25 @@ func listTestResultForVariant(featureGate string, jobVariant JobVariant) (*Testi
 			Host:   "sippy.dptools.openshift.org",
 			Path:   "api/tests",
 		}
-
-		latestRelease, err := getLatestRelease()
-		if err != nil {
-			return nil, fmt.Errorf("couldn't fetch latest release version: %w", err)
+		var release string
+		var err error
+		// if its not main branch, then use the ENV var to determine the release version
+		currentRelease := os.Getenv("PULL_BASE_REF")
+		if len(currentRelease) == 0 {
+			return nil, fmt.Errorf("unable to fetch the release version")
+		}
```

Comment:
This means you absolutely have to run the verify in the CI, and can't run it locally without setting this env var. I think drop this error, it doesn't change the flow too much and we end up falling back to where we were before

Reply:
makes sense